content/v2.0/backup-restore/backup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Back up data
+seotitle: Back up data in InfluxDB
+description: >
+  Use the `influx backup` command to back up all data and metadata stored in InfluxDB.
+menu:
+  v2_0:
+    parent: Back up & restore data
+weight: 101
+related:
+  - /v2.0/backup-restore/restore/
+  - /v2.0/reference/cli/influx/backup/
+draft: true
+---
+
+Use the [`influx backup` command](/v2.0/reference/cli/influx/backup/) to back up
+all data and metadata stored in InfluxDB.
+InfluxDB copies all data and metadata to a file set stored in a specified location
```

Comment:
copies data

Reply:
"all" is important here. The backup tool only supports full-backups. Not including "all" may leave room for people to think that incremental backups are a thing.

Reply:
Ah, I see. Perhaps we can explicate: Use....to run a full backup of InfluxDB? 
If folks want to do an incremental backup (hypothetical use case?), is there another way to do it? Note: To run an incremental backup, you must....

Reply:
Incremental backup isn't a thing yet, and it may be reserved for enterprise like it is in the 1.x line.